crates/autopilot/src/domain/auction/mod.rs

```diff
@@ -0,0 +1,37 @@
+use {
+    super::Order,
+    primitive_types::{H160, U256},
+    std::collections::BTreeMap,
+};
+
+pub mod order;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Auction {
+    /// The block that this auction is valid for.
+    /// The block number for the auction. Orders and prices are guaranteed to be
+    /// valid on this block.
```

Comment:
Are these simply copied from somewhere? If so, it might be better to "link" to them here rather than duplicating the comment which may get forgotten when we update it elsewhere. What is the most "canonical" representation of the model? Maybe still the one used by the model crate?

Reply:
added reference